services/app-api/src/domain-models/contractAndRates/convertContractWithRatesToHPP.ts

```diff
@@ -116,13 +116,7 @@ function convertContractWithRatesToFormData(
             } = rateRev.formData
 
             for (const additionalActuary of addtlActuaryContacts) {
-                if (
-                    !pkgAdditionalCertifyingActuaries.find((actuary) =>
-                        isEqualData(actuary, additionalActuary)
-                    )
-                ) {
-                    pkgAdditionalCertifyingActuaries.push(additionalActuary)
-                }
+                pkgAdditionalCertifyingActuaries.add(additionalActuary)
```

Comment:
Using `Set.add()` was causing duplicate additional actuaries. It might be good to pull in main again.